### PR TITLE
Update: Add banner

### PR DESCRIPTION
**Pull request by kaicataldo**
This PR implements a reusable banner we can use when we want to announce something.

Screenshot:
Screen Shot 2020-06-05 at 9 41 43 PM

### CODE REVIEW

File: src/styles/lib/banner.less [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.eslint-banner {
+  font-size: 24px;
+  font-weight: bold;
+  text-align: center;
+  margin: 0 auto;
+  background-color: #000000;
```
Comment (kaicataldo) [inline src/styles/lib/banner.less:6]: Sane defaults

File: _data/site.js [hunk 2]
```diff
@@ -17,6 +17,11 @@ function calculateUrl() {
 module.exports = {
     title: "ESLint - Pluggable JavaScript linter",
     description: "A pluggable and configurable linter tool for identifying and reporting on patterns in JavaScript. Maintain your code quality with ease.",
+    banner: {
+        text: null,
```
Comment (kaicataldo) [inline _data/site.js:21]: This won't render anything at the moment. I can update this with copy either in this PR or in a follow up once we finalize it.

Comment (nzakas) [reply _data/site.js:21]: Nice.

Comment (btmills) [reply _data/site.js:21]: Disclaimer: my understanding of the template syntax is limited. Would we be able to include links within this string? That’s the only improvement I could see wanting if that doesn’t work currently.

Comment (kaicataldo) [reply _data/site.js:21]: I wasn’t sure either, but I verified this does work!

**Pull request merged**